test-configs.yaml

```diff
@@ -146,6 +146,83 @@ test_plans:
             - 'defconfig+virtualvideo'
             - 'multi_v7_defconfig+virtualvideo'
 
+  uefi-arm64-gicv2:
+    rootfs: buildroot_ramdisk
+    params: {ctx_gicversion: "gic-version=2", ctx_cpu: "cortex-a57",
+             image_url_uefi: 'http://people.linaro.org/~ard.biesheuvel/kernelci/QEMU_EFI.fd-AARCH64-RELEASE-325ad6226099'
```

Comment:
I've copied these files on the staging storage server to test this on staging initially:
https://staging-storage.kernelci.org/images/uefi/325ad6226099/